docs/hub/unity-sentis.md

```diff
@@ -0,0 +1,66 @@
+# Using Sentis Models from Hugging Face
+
+[Unity Sentis](https://unity.com/products/sentis) is the inference engine that runs on Unity 2023 or above. 
+
+## Exploring Sentis Models in the Hub
+You will find `unity-sentis` models by filtering at the left of the [models page](https://huggingface.co/models?library=unity-sentis).
+
+All the Sentis models in the Hub come with code and instructions to easily get you started using the model in Unity. All Sentis models have been validated to work, so you can be sure they will run in Unity.
```

Comment:
How can we ensure validation going forward, when new models are (hopefully) submitted by users? Does it make sense to maintain a collection of validated models, if that's something useful for the community? In the meantime, how about slightly rewording to something like:

```suggestion
All the Sentis models in the Hub come with code and instructions to easily get you started using the model in Unity. All Sentis models under the `unity` namespace (for example, [unity/sentis-yolotinyv7](https://huggingface.co/unity/sentis-yolotinyv7)` have been validated to work, so you can be sure they will run in Unity.
```

Reply:
Yes that's more accurate.